### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by NavikaAgarwal** [recodehive/resume-pitch#98] [Q&A]
[Navika_Resume.pdf](https://github.com/user-attachments/files/17032090/Navika_Resume.pdf)
i'm a final year student with cse background, and looking for entry level job opportunities, so help me for the same
my linkedin profile - https://www.linkedin.com/in/navika-agarwal/

**Comment by sanjay-kv** [comment]
[nav.pdf](https://github.com/user-attachments/files/17044210/nav.pdf)
[ATS Version of CV (17).docx](https://github.com/user-attachments/files/17044212/ATS.Version.of.CV.17.docx)

**Comment by sanjay-kv** [comment]
sorry i missed it @NavikaAgarwal to address this on video

**Comment by NavikaAgarwal** [comment]
[Navika_Resume.pdf](https://github.com/user-attachments/files/17349058/Navika_Resume.pdf)
made the changes as suggested but will require help in making it up to date
